QModManager/VersionCheck.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Harmony;
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace QModManager
+{
+    internal static class VersionCheck
+    {
+        internal const string nexusmodsURL = "https://nexusmods.com/subnautica/mods/16";
+
+        internal static void Parse(string versionStr)
+        {
+            if (versionStr == null)
+            {
+                UnityEngine.Debug.Log("Could not get latest version!");
+                return;
+            }
+            Version version = new Version(versionStr);
+            if (version == null)
+            {
+                UnityEngine.Debug.Log("Could not get latest version!");
+                return;
+            }
+            if (!version.Equals(QMod.QModManagerVersion) && QModPatcher.erroredMods.Count <= 0) Dialog.Show($"There is a newer version of QModManager available: {version.ToString()}  (current version: {QMod.QModManagerVersion.ToString()})", 
+                () => Process.Start(nexusmodsURL), leftButtonText: "Download", blue: true);
+        }
+
+        internal const string VersionURL = "https://raw.githubusercontent.com/QModManager/QModManager/version-check/latest-version.txt";
```

Comment:
Any change that this could be pulled out into an external config file instead of being hard-coded into the source?

Reply:
The file will always be the same, and will not change. I don't see any reason to put it in an external config file

Reply:
Unless you control GitHub itself, there is no long term guarantee that user content will always be handled like this.  
Put a pin on this idea for a future revision.